app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  post.update(comments_counter: post.comments_counter + 1)
```

Comment:
- [ ] The comment update counter should be defined in a private  method for the comment model , kindly fix it 🙏 
example : 

```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```